clusterman/draining/queue.py

```diff
@@ -505,6 +505,7 @@ def host_from_instance_id(
 def process_queues(cluster_name: str) -> None:
     draining_client = DrainingClient(cluster_name)
     cluster_manager_name = staticconf.read_string(f"clusters.{cluster_name}.cluster_manager")
+    essential_drain_delay = staticconf.read_bool(f"clusters.{cluster_name}.essential_drain_delay", False)
```

Comment:
I would suggest calling it something like "always_delay_drain_processing", otherwise it may look like the field is expecting a timespan value.